mixer/test/client/env/v2_config.go

```diff
@@ -16,6 +16,8 @@
 package env
 
 import (
+	google_protobuf1 "github.com/gogo/protobuf/types"
```

Comment:
why use google_protobuf1?   can we shorten it as "gpb", similar to mpb

Reply:
api/mixer/v1/config/client/client_config.pb.go uses google_protobuf1. I have made it shorter.